CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Only listing significant user-visible, not internal code cleanups and minor bug
 * Improved communication between client and server.
 * Native jobs available in SparkUI
 * Improved native access to MongoDB
+* Fix multiples implementation of Slf4j
```

Comment:
I would say that it wasn't a major issue, so I wouldn't include it in the Changelog